mongo_connector/oplog_manager.py

```diff
@@ -668,16 +678,26 @@ def upsert_each(dm):
                             num_failed += 1
                         else:
                             raise
+                    if num % 10000 == 0:
+                        LOG.info("Upserted %d/%d docs from collection %s",
+                                 num + 1, total_docs, namespace)
+                if num is not None:
+                    LOG.info("Upserted %d/%d docs from collection %s",
+                             num + 1, total_docs, namespace)
             LOG.debug("Upserted %d docs" % num_inserted)
```

Comment:
Yeah. The total `num_inserted` can probably be removed now that we log per collection.